unitTest/matRad_unitTestTextManipulation.m

```diff
@@ -0,0 +1,60 @@
+function matRad_unitTestTextManipulation()
+
```

Comment:
dont you think it would make sense to factor this out into another function? I mean a function that gets three input arguments:

1. filename
2. string to be replaced
3. string 

and then you can just call this function three times from runMatRadTests

Reply:
Couldn't agree more!